check_trailing_spaces.sh

```diff
@@ -15,7 +15,7 @@ if [ "$files_with_trailing_spaces" ]
 then
     printf '\033[97;41mTrailing whitespaces detected:\033[0m\n'
     e=$(printf '\033')
-    echo "${files_with_trailing_spaces}" | sed -E "s/^\\.\\/([^:]+):([0-9]+):(.*[^\t ])?([\t ]+)$/${e}[0;31m - in ${e}[0;33m\\1${e}[0;31m at line ${e}[0;33m\\2\\n   ${e}[0;31m>${e}[0m \\3${e}[41;1m\\4${e}[0m/"
+    echo "${files_with_trailing_spaces}" | sed -E "s/^\\.\\/([^:]+):([0-9]+):(.*[^\\t ])?([\\t ]+)$/${e}[0;31m - in ${e}[0;33m\\1${e}[0;31m at line ${e}[0;33m\\2\\n   ${e}[0;31m>${e}[0m \\3${e}[41;1m\\4${e}[0m/"
```

Comment:
please, raise this change to https://github.com/PHP-CS-Fixer/trailing-whitespace-checker and let us see what would tests for it say ;)

Reply:
@keradus The tests pass: https://github.com/PHP-CS-Fixer/trailing-whitespace-checker/pull/10

Reply:
sounds great, original regex at PHP CS Fixer, that got fixed in your PR initially, was slightly different, and fixed version of it makes tests not pass, with new regex, it's passing indeed. great !